<ui version="4.0" >
 <comment>This file is part of the xTuple ERP: PostBooks Edition, a free and
open source Enterprise Resource Planning software suite,
Copyright (c) 1999-2014 by OpenMFG LLC, d/b/a xTuple.
It is licensed to you under the Common Public Attribution License
version 1.0, the full text of which (including xTuple-specific Exhibits)
is available at www.xtuple.com/CPAL.  By using this software, you agree
to be bound by its terms.</comment>
 <class>taxAssignments</class>
 <widget class="QWidget" name="taxAssignments" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>List Tax Assignments</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5" >
   <item row="0" column="0" >
    <layout class="QGridLayout" name="gridLayout_4" >
     <property name="margin" >
      <number>12</number>
     </property>
     <property name="spacing" >
      <number>12</number>
     </property>
     <item row="0" column="0" >
      <layout class="QHBoxLayout" name="horizontalLayout_2" >
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" >
         <item>
          <widget class="QGroupBox" name="_showZoneGroup" >
           <property name="title" >
            <string>Tax Zones</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2" >
            <item row="0" column="0" >
             <layout class="QHBoxLayout" >
              <property name="spacing" >
               <number>6</number>
              </property>
              <property name="margin" >
               <number>0</number>
              </property>
              <item>
               <widget class="QRadioButton" name="_showAll" >
                <property name="text" >
                 <string>All</string>
                </property>
                <property name="checked" >
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <spacer>
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0" >
                 <size>
                  <width>161</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item row="1" column="0" >
             <layout class="QHBoxLayout" >
              <property name="spacing" >
               <number>6</number>
              </property>
              <property name="margin" >
               <number>0</number>
              </property>
              <item>
               <widget class="QRadioButton" name="_showOne" >
                <property name="text" >
                 <string>Specific Tax Zone:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="XComboBox" name="_taxZone" >
                <property name="enabled" >
                 <bool>false</bool>
                </property>
                <property name="allowNull" >
                 <bool>true</bool>
                </property>
                <property name="nullStr" >
                 <string>Any</string>
                </property>
                <property name="type" >
                 <enum>XComboBox::TaxZones</enum>
                </property>
               </widget>
              </item>
              <item>
               <spacer>
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0" >
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="_showTypeGroup" >
           <property name="title" >
            <string>Tax Types</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3" >
            <item row="0" column="0" >
             <layout class="QHBoxLayout" name="_2" >
              <property name="spacing" >
               <number>6</number>
              </property>
              <property name="margin" >
               <number>0</number>
              </property>
              <item>
               <widget class="QRadioButton" name="_showAllTypes" >
                <property name="text" >
                 <string>All</string>
                </property>
                <property name="checked" >
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <spacer>
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0" >
                 <size>
                  <width>161</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item row="1" column="0" >
             <layout class="QHBoxLayout" name="_3" >
              <property name="spacing" >
               <number>6</number>
              </property>
              <property name="margin" >
               <number>0</number>
              </property>
              <item>
               <widget class="QRadioButton" name="_showOneType" >
                <property name="text" >
                 <string>Specific Tax Type:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="XComboBox" name="_taxType" >
                <property name="enabled" >
                 <bool>false</bool>
                </property>
                <property name="allowNull" >
                 <bool>true</bool>
                </property>
                <property name="nullStr" >
                 <string>Any</string>
                </property>
                <property name="type" >
                 <enum>XComboBox::TaxTypes</enum>
                </property>
               </widget>
              </item>
              <item>
               <spacer>
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0" >
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_3" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>78</width>
           <height>81</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" >
         <property name="spacing" >
          <number>0</number>
         </property>
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="_close" >
           <property name="text" >
            <string>Close</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>20</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="0" >
      <widget class="QFrame" name="_frame" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape" >
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>0</number>
        </property>
        <item row="0" column="0" >
         <layout class="QVBoxLayout" name="verticalLayout" >
          <property name="spacing" >
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>Tax Assignments:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <property name="spacing" >
             <number>6</number>
            </property>
            <property name="margin" >
             <number>0</number>
            </property>
            <item>
             <widget class="XTreeWidget" name="_taxass" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
                <horstretch>0</horstretch>
                <verstretch>5</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" >
              <property name="spacing" >
               <number>0</number>
              </property>
              <property name="margin" >
               <number>0</number>
              </property>
              <item>
               <widget class="QPushButton" name="_new" >
                <property name="text" >
                 <string>New</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="_edit" >
                <property name="enabled" >
                 <bool>false</bool>
                </property>
                <property name="text" >
                 <string>Edit</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="_view" >
                <property name="enabled" >
                 <bool>false</bool>
                </property>
                <property name="text" >
                 <string>View</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="_delete" >
                <property name="enabled" >
                 <bool>false</bool>
                </property>
                <property name="text" >
                 <string>Delete</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer>
                <property name="orientation" >
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0" >
                 <size>
                  <width>20</width>
                  <height>0</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="5" margin="5" />
 <customwidgets>
  <customwidget>
   <class>XComboBox</class>
   <extends>QComboBox</extends>
   <header>xcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>XTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>xtreewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>_close</sender>
   <signal>clicked()</signal>
   <receiver>taxAssignments</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>737</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel" >
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_showOne</sender>
   <signal>toggled(bool)</signal>
   <receiver>_taxZone</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>115</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel" >
     <x>293</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_showOneType</sender>
   <signal>toggled(bool)</signal>
   <receiver>_taxType</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>350</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel" >
     <x>446</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
